packages/api/src/controllers/stream.ts

```diff
@@ -910,7 +899,8 @@ app.put(
           const ingest = ((await req.getIngest()) ?? [])[0]?.base;
           const recordingUrl = getRecordingUrl(ingest, session);
           const mp4Url = getRecordingUrl(ingest, session, true);
-          req.queue.delayedEmit(
+          req.queue.delayedPublish(
```

Comment:
should add `await` to every call to `publish` - or else exception inside it will crash whole node with `unhandled promise rejection`